samples/03.live-inking/src/simulation.ts

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
in this case, does simulation mean testing? Or does it have to do with performance improvements? Simulation makes me think of test simulators haha.

Reply:
I will actually remove all this as the testing is now pushed to 31.inking-tests

This simulation was a way to simulate multiple users drawing at the same time, by opening multiple browser windows on the same machine.